### PR TITLE
Updating s3 docs for Automate and A2HA

### PR DESCRIPTION
**Pull request by vinay-satish**
### Description

[Please describe what this change achieves]

### Issues Resolved

[List any existing issues this PR resolves, or any Discourse or
StackOverflow discussions that are relevant]

### Check List

- [ ] New functionality includes tests
- [ ] All buildkite tests pass
- [ ] Full omnibus build and tests in buildkite pass
- [ ] All commits have been signed-off for the Developer Certificate of Origin. See <https://github.com/chef/chef/blob/main/CONTRIBUTING.md#developer-certification-of-origin-dco>
- [ ] PR title is a worthy inclusion in the CHANGELOG

### CODE REVIEW

File: docs-chef-io/content/server/_index.md [hunk 1]
```diff
@@ -229,6 +229,13 @@ modification when using an external S3 provider:
 </tbody>
 </table>
 
+{{< note >}}
+
+For automate or A2HA environment the `chef-server.rb` will not be present. In that case, we need to update
+the `s3_url`, `s3_external_url` and `s3_platform_bucket_name` with appropriate value in the `sys.config`
```
Comment (vinay-satish) [inline docs-chef-io/content/server/_index.md:235]: Add the location - '$(hab pkg path chef/automate-cs-oc-erchef)/config/sys.config'

**Comment by IanMadd** [comment]
Why should this be in the Server docs? This seems like content that should be in an Automate config doc. I also don't see any of those config settings in the server docs or anywhere else.

**Comment by IanMadd** [comment]
Maybe I'm missing something.

First, we shouldn't be talking about A2HA in Server docs. 

None of those settings are referenced in the Server docs, so if users can and should modify or set them in Server, then we should document what they are first. 

From the context here, it looks like something that only applies to Automate users, so this would fall under a doc for configuring Server to work with Automate in the Automate docs.

If chef-server.rb isn't present, then what is the alternative? Why do they care about sys.config if their issue is the chef-server.rb. The relationship between the two isn't clear here. Can the settings that normally fall in the chef-server.rb be passed from the sys.config, or is that file stored on S3 and the path is in the sys.config?

Also, it looks like you're not specifying a location, but a command to run to return a file path.

**Comment by IanMadd** [comment]
Closing because there hasn't been a response to my questions.

**Pull request closed without merging**